txframework/pom.xml

```diff
@@ -219,7 +219,7 @@
              <jdk>[11,)</jdk>
           </activation>
           <properties>
-             <jvm.args.modular>--add-modules=java.se</jvm.args.modular>
+             <jvm.args.modular>--add-modules=java.se --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.time=ALL-UNNAMED --add-opens=java.naming/javax.naming=ALL-UNNAMED</jvm.args.modular>
```

Comment:
@mayankkunwar please can you add a comment indicating why these settings are needed?

Reply:
okay